src/responder/kcm/secrets/secrets.c

```diff
@@ -57,6 +57,42 @@ static char *local_dn_to_path(TALLOC_CTX *mem_ctx,
                               struct ldb_dn *basedn,
                               struct ldb_dn *dn);
 
+static int talloc_mem_destructor(uint8_t *data)
+{
+    if (data != NULL) {
+        sss_erase_talloc_mem_securely(data);
+    }
+
+    return 0;
+}
+
+static void local_db_erase_message_securely(struct ldb_message *msg, const char *attr)
```

Comment:
This function only touches `ldb_message` kept in memory, right?
Imo, function name is misleading as it suggest "local db" (i.e. files) operations...

Reply:
Agreed. I was considering changing the prefix.